library/ssl_misc.h

```diff
@@ -2682,4 +2682,25 @@ int mbedtls_ssl_session_set_hostname( mbedtls_ssl_session *session,
                                       const char *hostname );
 #endif
 
+#if defined(MBEDTLS_SSL_PROTO_TLS1_3) && defined(MBEDTLS_SSL_SESSION_TICKETS)
+static inline uint8_t mbedtls_ssl_tls13_session_get_ticket_flags(
```

Comment:
I think `ticket_flags` field is TLS 1.3 specific.

> Otherwise I would not bother with using a short type uint8_t for the flags I would just use unsigned int

I found there is a definition of `mbedtls_ssl_tls13_ticket_flags`, but it is not used anywhere. Do you think it's more proper to be used for the ticket_flags?

Reply:
> I think `ticket_flags` field is TLS 1.3 specific.
> 

Yes I know but as ticket flags is quite generic and may make sense in future versions of TLS or even in TLS 1.2 with new extensions I prefer just `mbedtls_ssl_session_get_ticket_flags`.

Reply:
> I found there is a definition of `mbedtls_ssl_tls13_ticket_flags`, but it is not used anywhere. Do you think it's more proper to be used for the ticket_flags?

If `mbedtls_ssl_tls13_ticket_flags` is not used I would just remove it and change the signature of `mbedtls_ssl_session_get_ticket_flags` to
`static inline unsigned int mbedtls_ssl_tls13_session_get_ticket_flags(  mbedtls_ssl_session *session, unsigned int flags )`. That the compiler does not have to bother with the uint8_t type which may lead to additional instructions.